.gitignore

```diff
@@ -32,4 +32,9 @@ lerna-debug.log*
 !.vscode/settings.json
 !.vscode/tasks.json
 !.vscode/launch.json
-!.vscode/extensions.json
\ No newline at end of file
+!.vscode/extensions.json
+
+#.env
+**/.env
+**/development.env
```

Comment:
Always use .env.* for environment variables and not the other way around. You environment variable file should start with something like this 

`.env.development`, `.env.production` ...
This way your .gitignore would be simple as well.  (.env*)